src/build/app/settings.rs

```diff
@@ -417,14 +416,15 @@ pub enum AppSettings {
     ///     .setting(AppSettings::ArgsNegateSubcommands)
     /// # ;
     /// ```
-    /// [subcommands]: ./struct..html
-    /// [argument]: Arg
+    ///
```

Comment:
Also, please remove these empty lines.

Reply:
left them in because things felt too cramped... do you insist?

Reply:
I would like it to be consistent. I actually like the new line, but please do that for all the doc blocks you are touching instead of only some.